magetab_validation.py

```diff
@@ -20,25 +19,28 @@
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('idf',
-                        help="name of MAGE-TAB IDF file")
-
+                        help="Path to the MAGE-TAB IDF file")
+    parser.add_argument('-d', "--data_dir", default="",
+                        help="Path to the directory with SDRF and data files")
+    parser.add_argument('-v', '--verbose', action='store_const', const=10, default=20,
+                        help="Option to output detailed logging (debug level).")
     args = parser.parse_args()
 
-    return args.idf
+    return args.idf, args.data_dir, args.verbose
```

Comment:
Why not return the complete args object here instead of having to change this every time that you have more args?

Reply:
Thanks for pointing this out. Changed as suggested. Although I still use the variable unpacking for now.